src/reducer.ts

```diff
@@ -99,8 +101,9 @@ export default combineReducers<AppState>({
     features: featuresReducer,
     veiledergrupperLagretFilter: veiledergrupperLagretFilterReducer,
     mineFilter: mineFilterReducer,
-    mineFilterMinOversikt: named(ListevisningType.minOversikt, mineFilterUI),
-    mineFilterEnhetensOversikt: named(ListevisningType.enhetensOversikt, mineFilterUI),
+    lagretFilterMinOversikt: named(ListevisningType.minOversikt, mineFilterUI),
```

Comment:
same here, no?
lagretFilterMinOversikt => mineFilterMinOversikt
lagretFilterEnhetensOversikt => mineFilterEnhetensOversikt
lagretFilterVeilederOversikt => mineFilterVeilederOversikt

and:
mineFilterUI => lagretFilterUI